### PR TITLE
Interactive mode!

### PR DESCRIPTION
**Pull request by hackergrrl**
A few commits to integrate noffle/place-geo-marker with geopkg, letting the user use `geopkg interactive` to both place a new marker, or modify the location of an existing marker.

Addresses #1.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -48,20 +51,14 @@ function update () {
 }
 
 function open () {
-  try {
-    var targetPkg = require(path.join(process.cwd(), 'package'))
-  } catch (e) {
+  var currentCoords = geopkg.getPackageCoords()
+  if (currentCoords) {
+    return geopkg.openMaps(currentCoords, _done)
+  } else {
```
Comment (watson) [inline index.js]: This `else` block will run both if package.json couldn't be found but also if the package.json didn't contain any coordinates - so the error shown is not always correct.

Previously the code differentiated between showing an error for when package.json wasn't found and when the package.json didn't contain any coordinates.

Comment (hackergrrl) [inline index.js]: Oh shoot, I missed that subtle (but very important) distinction.

**Comment by hackergrrl** [comment]
Anything else I can do here?

**Comment by watson** [comment]
@noffle ups, was held up with work so totally forgot about this. It's now out on npm as v2.2.0. I've added you as a collaborator as well :smiley:

**Comment by hackergrrl** [comment]
\o/

**Pull request merged**